app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(" ")
+        else:
+            if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+                raise OutdatedVaccineError(" ")
+
+        if not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError(" ")

```

Comment:
The error message for `NotWearingMaskError` is not descriptive. According to the checklist, you should use descriptive error messages.